uips/uip-7.md

```diff
@@ -0,0 +1,146 @@
+| uip | 007 |
+| - | - |
+| title | Transparent Addresses for External Compatibility |
+| description | A 32-byte address format for improved interoperability with external systems |
+| author | Jennifer Helsby ([@redshiftzero](https://github.com/redshiftzero)), Henry de Valence ([@hdevalence](https://github.com/hdevalence)), Lúcás Meier ([@cronokirby](https://github.com/cronokirby)) |
+| discussions-to | <https://forum.penumbra.zone/t/pre-uip-transparent-addresses/140> |
+| status | Draft |
+| type | Standards |
+| consensus | Yes |
+| created | 2024-12-10 |
+
+## Abstract
+
+This UIP introduces transparent addresses ("t-addresses"), a 32-byte Bech32 address format designed for maximum compatibility with external systems while maintaining convertability to full Penumbra addresses.
+
+## Motivation
+
+Recent compatibility issues with external systems (e.g., Noble's USDC integration) have highlighted the need for a maximally compatible address format. While the correct long-term solution is full support for Penumbra's native addresses, a shorter address format can serve as an immediate solution and future-proof escape hatch for similar issues. Penumbra's native addresses have two major differences from most other addresses in the Cosmos ecosystem, in that:
+
+1. They are longer: 144 bytes.
+2. They use Bech32m encoding, instead of Bech32.
```

Comment:
Would be useful to follow this list with a one sentence descirption that t-addrs are 32 bytes Bech32 encoded